examples/tutorials/inset.py

```diff
@@ -0,0 +1,109 @@
+"""
+Adding an inset to the figure
+=============================
+
+To plot an inset figure inside another larger figure, we can use the
+:meth:`pygmt.Figure.inset` method. After a large figure has been created,
+call ``inset`` using a ``with`` statement, and new plot elements will be
+added to the inset figure instead of the larger figure.
+"""
```

Comment:
Choose the best image as the thumbnail, see https://github.com/GenericMappingTools/pygmt/pull/826 and #819.
```suggestion
"""
# sphinx_gallery_thumbnail_number = 4
```